apps/meteor/app/lib/server/functions/setUserActiveStatus.ts

```diff
@@ -105,6 +107,7 @@ export async function setUserActiveStatus(userId: string, active: boolean, confi
 	if (active === false) {
 		await Users.unsetLoginTokens(userId);
 		await Rooms.setDmReadOnlyByUserId(userId, undefined, true, false);
+		void notifyListenerOnRoomsChanges(await Rooms.find({ uids: { $size: 2, $in: [userId] }, t: 'd' }).toArray());
```

Comment:
pls move this query to a model method:
```suggestion
		void notifyListenerOnRoomsChanges(await Rooms.findDMsByUserId(userId).toArray());
```